README.md

```diff
@@ -1,8 +1,44 @@
+[![Join Slack](https://img.shields.io/badge/slack-join%20chat-4a154b)](https://join.slack.com/t/codeforpoznan/shared_invite/zt-8a7u52j8-yqB01C2YgYF4Lvd1pFM_jw)
+
+## General Overview of the Project
+This is internal project maintained by our community for our comminity. 
+
+First, we maintain the landing page that contains information about 
+our community and the projects we carry out for other organizations 
+that help them do good more easily or efficiently. They don't have much money 
+and software development is expensive. We simply try to bridge that gap 
+with our voluntary work to balance the scales.
+
+Second we develop the intranet for our community. We try to create the tools 
+which can help us organize ourselves better by storing the relevant information
+in one place, streamlining our practices and managing our responsibilities. 
+If you want to learn the general direction we try to accomplish, here you'll 
+find the general description of features we pursue at the moment:
+[Features we currently pursue](https://github.com/CodeForPoznan/codeforpoznan.pl_v3/issues?q=is%3Aopen+is%3Aissue+label%3AEpic)
+
+### Wanna help us develop the product?
+* [Content - tasks that need some writing](https://github.com/CodeForPoznan/codeforpoznan.pl_v3/issues?q=is%3Aopen+is%3Aissue+label%3A%22content+needed%22+sort%3Acreated-asc)
+* [Design - tasks that could use UX experise]()
+
+## How to get started
+1. Join codeforpoznan_v3 channel on [our slack](https://join.slack.com/t/codeforpoznan/shared_invite/zt-8a7u52j8-yqB01C2YgYF4Lvd1pFM_jw) - this is the place where we discuss all the issues and help each other by sharing tips and knowledge. You can also ping @Otis in the private channel if you need any help.
+2. Fork the repository onto your github account, choose the location on your computer where you want to keep it  and ```git clone PASTE URL``` in the terminal, using the URL link from your copy of the repo - this way any commit you push will affect only your fork (so you can break anything you like).
+3. Install docker according to the instruction you'll fine [here](https://docs.docker.com/engine/install/) to run and develop the app locally on your computer. Once you do it, got to the location of the repo on your computer and run command ```make start``` in the terminal (add ```sudo``` before the command in case of permission troubles) you'll be able to access the frontend on ```localhost:8080``` and push requests to backend on ```localhost:5000```.
+4. Choose a task from one if the lists below and leave a comment that you're gonna work on it.
+5. We like to keep ```master``` branch clean so create a new branch ```git checkout -b BRANCH NAME``` - name the branch descriptively e.g. ```update_something``` so that you know what's there. Then, switch branch ```git checkout update_something``` and start developing!
```

Comment:
I'd add the convention to add prefixes in branches and commit messages, eg. if you work on ticket CFP-123:
- branch: `CFP-123_update_something`
- commit: `[CFP-123] Update something`